docs/resources/Guild_Event.md

```diff
@@ -0,0 +1,100 @@
+| user_count?          | integer                                                                                       | users subscribed to the event                                                             |
+
+###### Guild Event Entity Types
+
+| Type      | Value |
+| --------- | ----- |
+| SCHEDULED | 1     |
+| ACTIVE    | 2     |
+| COMPLETED | 3     |
+| CANCELED  | 4     |
+
+###### Guild Event Status
+
+| Type           | Value |
+| -------------- | ----- |
+| NONE           | 0     |
+| STAGE_INSTANCE | 1     |
+
+###### Guild Event Entity Metadata
+
+| Field        | Type                | Description                       |
+| ------------ | ------------------- | --------------------------------- |
+| speaker_ids? | array of snowflakes | the speakers of the stage channel |
+
+## List Guild Events for Guild % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/events
+
+Returns a list of [Guild Events](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) in the guild.
+
+###### Query String Params
+
+| Field            | Type    | Description                   |
+| ---------------- | ------- | ----------------------------- |
+| with_user_count? | boolean | users subscribed to the event |
+
+## Create Guild #vent % POST /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/events
+
+Create an event in the guild. Returns a [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) object on success.
+
+###### JSON Params
+
+| Field                | Type                                                                                  | Description                       |
+| -------------------- | ------------------------------------------------------------------------------------- | --------------------------------- |
+| channel_id?          | snowflake                                                                             | the stage channel id of the event |
+| name                 | string                                                                                | the name of the event             |
+| privacy_level        | [privacy level](#DOCS_RESOURCES_STAGE_INSTANCE/stage-instance-object-privacy-level)   | the privacy level of the event    |
+| scheduled_start_time | ISO8601                                                                               | the time to schedule the event    |
+| description?         | string                                                                                | the description of the event      |
+| entity_type?         | [entity type](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object-guild-event-entity-type) | event entity type                 |
+
+## List Guild Events % GET /guild-events
+
+Returns a list of [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) objects available to the bot.
```

Comment:
looks like bots cant use this endpoint

Reply:
Only if you pass an ID afterwards

Reply:
yes, that is a separate endpoint documented below